contracts/contracts/BoosterLite.sol

```diff
@@ -0,0 +1,572 @@
+import "./Interfaces.sol";
+import "@openzeppelin/contracts-0.6/math/SafeMath.sol";
+import "@openzeppelin/contracts-0.6/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts-0.6/utils/Address.sol";
+import "@openzeppelin/contracts-0.6/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts-0.6/utils/ReentrancyGuard.sol";
+
+interface ICoordinator {
+    function queueNewRewards(address, uint256) external payable;
+}
+
+/**
+ * @title   BoosterLite
+ * @author  ConvexFinance -> AuraFinance
+ * @dev     A lite version of the original Booster for use on sidechains
+ */
+contract BoosterLite is ReentrancyGuard {
+    using SafeERC20 for IERC20;
+    using Address for address;
+    using SafeMath for uint256;
+
+    address public crv;
+
+    uint256 public lockIncentive = 825; //incentive to crv stakers
+    uint256 public stakerIncentive = 825; //incentive to native token stakers
+    uint256 public earmarkIncentive = 50; //incentive to users who spend gas to make calls
+    uint256 public platformFee = 0; //possible fee to build treasury
+    uint256 public constant MaxFees = 4000;
+    uint256 public constant FEE_DENOMINATOR = 10000;
+
+    address public owner;
+    address public feeManager;
+    address public poolManager;
+    address public immutable staker;
+    address public minter;
+    address public rewardFactory;
+    address public stashFactory;
+    address public tokenFactory;
+    address public treasury;
+    address public rewards; //cvxCrv rewards(crv)
+    bool public isShutdown;
+
+    struct PoolInfo {
+        address lptoken;
+        address token;
+        address gauge;
+        address crvRewards;
+        address stash;
+        bool shutdown;
+    }
+
+    //index(pid) -> pool
+    PoolInfo[] public poolInfo;
+    mapping(address => bool) public gaugeMap;
+
+    event Deposited(address indexed user, uint256 indexed poolid, uint256 amount);
+    event Withdrawn(address indexed user, uint256 indexed poolid, uint256 amount);
+
+    event PoolAdded(address lpToken, address gauge, address token, address rewardPool, address stash, uint256 pid);
+    event PoolShutdown(uint256 poolId);
+
+    event OwnerUpdated(address newOwner);
+    event FeeManagerUpdated(address newFeeManager);
+    event PoolManagerUpdated(address newPoolManager);
+    event FactoriesUpdated(address rewardFactory, address stashFactory, address tokenFactory);
+    event RewardContractsUpdated(address rewards);
+    event FeesUpdated(uint256 lockIncentive, uint256 stakerIncentive, uint256 earmarkIncentive, uint256 platformFee);
+    event TreasuryUpdated(address newTreasury);
+
+    /**
+     * @dev Constructor doing what constructors do. It is noteworthy that
+     *      a lot of basic config is set to 0 - expecting subsequent calls setFeeManager etc.
+     * @param _staker          VoterProxyLite (locks the crv and adds to all gauges)
+     */
+    constructor(address _staker) public {
+        staker = _staker;
+        isShutdown = false;
```

Comment:
redundant setting this in the constructor.

Reply:
Explicitly settings these is a pattern convex and others like to follow so I think it is fine to leave these, they likely get optimised out anyway.